ApplicationLibCode/Commands/ExportCommands/RicExportSectorModelUi.cpp

```diff
@@ -134,7 +146,7 @@ void RicExportSectorModelUi::defineUiOrdering( QString uiConfigName, caf::PdmUiO
     // must be kept in sync with the page names defined in the constructor
     CAF_ASSERT( m_pageNames.size() == 5 );
 
-    if ( uiConfigName == m_pageNames[0] )
+    if ( uiConfigName == m_pageNames[0] ) // source data
```

Comment:
Consider adding functions instead of lookup in a vector.

    if ( uiConfigName == sourceDataName() )